<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamMainWindowClass</class>
 <widget class="QMainWindow" name="StreamMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stream_start_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stream_stop_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Stream</string>
    </property>
   </widget>
   <widget class="QLabel" name="ip_adress_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Adress: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="watch_start_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Watch</string>
    </property>
   </widget>
   <widget class="DisplayGLWidget" name="displayGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="listen_port_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="stream_port_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stream_port_line">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10001</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_adress_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="listen_port_line">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10001</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DisplayGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>DisplayGLWidget.h</header>
   <slots>
    <slot>onStartButtonClicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="StreamMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stream_start_btn</sender>
   <signal>clicked()</signal>
   <receiver>StreamMainWindowClass</receiver>
   <slot>onStartButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stream_stop_btn</sender>
   <signal>clicked()</signal>
   <receiver>StreamMainWindowClass</receiver>
   <slot>onLeaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>watch_start_btn</sender>
   <signal>clicked()</signal>
   <receiver>StreamMainWindowClass</receiver>
   <slot>onWatchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartButtonClicked()</slot>
  <slot>onLeaveButtonClicked()</slot>
  <slot>onWatchButtonClicked()</slot>
 </slots>
</ui>
